### DISCUSSION TITLE
Can i use options api with petite vue?

### DISCUSSION THREAD

**Discussion by ricky11** [vuejs/petite-vue#88] [Q&A] [answered]
I am use to the options api with vue 2.x,

petite-vue seems to be using the composition api, can i use the options api with petite vue?

thanks

**Comment by niutech** [comment, marked as answer]
Petite Vue is using options API, just simplified. There is no `{data: {}, methods: {}, computed: {}}`, but just:
```
{
    // data
    count: 0,
    // computed
    get plusOne() {
      return this.count + 1
    },
    // methods
    increment() {
      this.count++
    }
  }
```
